test/ProgressOnderwijsUtils.Tests/MetaObjectBulkCopyTest.cs

```diff
@@ -152,6 +157,71 @@ public void BulkCopySupportsCumputedColumn()
             PAssert.That(() => fromDb.Computed);
         }
 
+        sealed class IncludingIdentityColumn : ValueBase<IncludingIdentityColumn>, IMetaObject, IPropertiesAreUsedImplicitly
+        {
+            public int Id { get; set; }
+            public int AnIdentity { get; set; }
+            public string Bla { get; set; }
+        }
+
+        sealed class ExcludingIdentityColumn : ValueBase<ExcludingIdentityColumn>, IMetaObject, IPropertiesAreUsedImplicitly
+        {
+            public int Id { get; set; }
+            public string Bla { get; set; }
+        }
+
+        [Fact]
+        public void BulkCopyIgnoresIdentityColumns()

```

Comment:
Added test here for bulk inserting into tables with identities ( in the default no-KeepIdentity mode)